openapi.yml

```diff
@@ -977,12 +978,13 @@ components:
       properties:
         sourceId:
```

Comment:
I wonder if this repetition means we ought to extract source id as its own component which can then be referred to via $ref wherever needed?

Reply:
done